duckdns/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.21.0
+
+- Call Dehydrator per-domain or alias due to limitation in DuckDNS which can only handle a single TXT record at a time
+- Output text to show the domain or alias being processed
+- Remove filtering of the domain = alias, otherwise, aliases are not getting renewed
+- DuckDNS NameServer TTL is observed at 60s, so, we need to wait longer than that
```

Comment:
If you want to keep the observation as an information for future devs, add it as a comment before timeout command. For the logs, simply state what changed:

```suggestion
- Increase DuckDNS name server TTL to 120s
```